src/clj_github/httpkit_client.clj

```diff
@@ -11,14 +11,20 @@
 (defn get-installation-token [{:keys [token-fn]}]
   (token-fn))
 
+(defn- append-url-path [baseurl path]
```

Comment:
Wdyt about adding some unit tests for these cases
```
(append-url-path "github.com" "path")
(append-url-path "github.com/" "path")
(append-url-path "github.com" "/path")
```

Reply:
@gabrielgiussi this is a private function and implicitly tested in https://github.com/nubank/clj-github/pull/10/files#diff-3d82479c9bb9ffd50aac8b0bc3dacf8754ae47fb4c95de6794a0f7d25e870a4cR27. Also the github base url is defined as a constant, I don't think adding these tests add much value.